src/controller/CHIPCluster.cpp

```diff
@@ -42,6 +42,37 @@ CHIP_ERROR ClusterBase::Associate(DeviceProxy * device, EndpointId endpoint)
     return err;
 }
 
+CHIP_ERROR ClusterBase::Associate(GroupId groupId, DeviceProxy * device)
+{
+    CHIP_ERROR err = CHIP_NO_ERROR;
+    // TODO: Check if the device supports mCluster at the requested endpoint
```

Comment:
This comment makes no sense in a group context.  Please take it out.

Reply:
Done